dashbuilder/branch-overrides.yaml

```diff
@@ -0,0 +1,35 @@
+schema_version: 1
+
+# Purpose: Branch overrides are for building the current state of the branch,
+#          along with other modules at specific reference points. Neither the
+#          branch nor the modules it references are guaranteed to be unchanging.
+#   Usage: cekit build --overrides-file branch-overrides.yaml
+
+modules:
+  repositories:
+    - name: cct_module
+      git:
+        url: https://github.com/jboss-openshift/cct_module.git
+        ref: 0.39.0
+    - name: jboss-eap-modules
+      git:
+        url: https://github.com/jboss-container-images/jboss-eap-modules.git
+        ref: EAP_733_CR2
+    - name: jboss-eap-7-image
+      git:
+        url: https://github.com/jboss-container-images/jboss-eap-7-image.git
+        ref: EAP_733_CR2
+    - name: wildfly-cekit-modules
+      git:
+        url: https://github.com/wildfly/wildfly-cekit-modules.git
+        ref: 0.18.6
+    - name: rhpam-7-image
+      path: /data/dev/sources/rhpam-7-image
```

Comment:
Why is the path given here, instead of the git url?

Reply:
draft yet, as the all other changes are local yet, need to use it, will update when the PR is ready.

Reply:
Sorry, didn't notice :p. Will check when it is ready :)

Reply:
@Kaitou786 now it is ready for review :)